app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -1,18 +1,222 @@
 package com.pvp.app.ui.screen.task
 
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Card
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.pvp.app.model.MealTask
+import com.pvp.app.model.SportActivity
+import com.pvp.app.model.SportTask
 import com.pvp.app.model.Task
+import java.time.Duration
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+// Parses Duration object to a string
+// Format <HH> h <mm> min
+fun getDurationString(duration: Duration): String {
+    val hours = duration.toHours()
+    val minutes = duration.minusHours(hours).toMinutes()
+
+    return buildString {
+        if (hours > 0) append("$hours h")
+        if (minutes > 0) append("${if (hours > 0) " " else ""}$minutes min")
+    }
+}
+@Composable
+fun MealTaskBoxBody(
+    mealTask: MealTask
+){
```

Comment:
Single spacing before opening curly brackets should be added. `Ctrl+Alt+L` hotkey should format it automatically.